### PR TITLE
fix: Adding error message to git hook when commitlint not installed.

### PR DESCRIPTION
**Pull request by danielmoy-google**
Unclear if we'd want this somewhere else.

### CODE REVIEW

File: tools/git/commit-msg.sh [hunk 1]
```diff
@@ -15,4 +15,8 @@
 # limitations under the License.
 
 # Check commit message: https://www.conventionalcommits.org/en/v1.0.0-beta.2/
+if ! which commitlint; then
+  echo "FAILURE: commitlint not installed"
+  exit 1
+fi
```
Comment (schroederc) [inline tools/git/commit-msg.sh]: There's now no need for the `command -v commitlint &>/dev/null` below.

File: tools/git/commit-msg.sh [hunk 2]
```diff
@@ -15,4 +15,8 @@
 # limitations under the License.
 
 # Check commit message: https://www.conventionalcommits.org/en/v1.0.0-beta.2/
+if ! which commitlint; then
```
Comment (schroederc) [inline tools/git/commit-msg.sh]: Pipe stdout/stderr to /dev/null: `! which commitlint &>/dev/null`

File: tools/git/commit-msg.sh [hunk 3]
```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 
 # Check commit message: https://www.conventionalcommits.org/en/v1.0.0-beta.2/
-if ! which commitlint; then
+if ! which commitlint &>/dev/null; then
   echo "FAILURE: commitlint not installed"
```
Comment (schroederc) [inline tools/git/commit-msg.sh]: Sorry for the late comment, but this should probably go to stderr: `>&2`

Comment (danielmoy-google) [reply tools/git/commit-msg.sh]: done

**Pull request merged**